lib/logstash/filters/ruby.rb

```diff
@@ -66,6 +66,9 @@ def register
     else
       @logger.fatal("You must either use an inline script with the \"code\" option or a script file using \"path\".")
     end
+  rescue ScriptError, SyntaxError => e
```

Comment:
sorry I did not realize there's a `ScriptError` in scope, what I really wanted is:
```suggestion
  rescue ::ScriptError => e # SyntaxError, LoadError etc.
```